ansible_base/rest_filters/utils.py

```diff
@@ -44,6 +44,8 @@ def get_fields_from_path(model, path, treat_jsonfield_as_text=True):
 
         if name in getattr(model, 'PASSWORD_FIELDS', ()):
             raise PermissionDenied(_('Filtering on password fields is not allowed.'))
+        elif name in getattr(model, 'encrypted_fields', ()):
+            raise PermissionDenied(_('Filtering on encrypted fields is not allowed.'))
```

Comment:
In thinking about this I don't like the origional message we had in here about password fields and I'm thinking we won't want to add multiple message types if we wanted to prevent filtering on some other kind of field. What if we return a generic message like this:

```suggestion
        if name in getattr(model, 'PASSWORD_FIELDS', ()) or name in getattr(model, 'encrypted_fields', ()):
            raise PermissionDenied(_('Filtering on field %(name)s is not allowed.') % {'name': name})
```

If we do this, we will need to change the test as well.

Reply:
Do you think we have to worry about this change affecting Ansible Connector's test results? Does not seem like it will, but i figured I would check with you as well.

Reply:
Applied the requested changes, and modified test case to match.